common/src/test/scala/uk/ac/wellcome/test/utils/SQSLocal.scala

```diff
@@ -16,12 +24,28 @@ trait SQSLocal extends Suite with BeforeAndAfterEach {
       new EndpointConfiguration(s"http://localhost:9324", "localhost"))
     .build()
 
-  val idMinterQueue = "es_id_minter_queue"
-  val idMinterQueueUrl = sqsClient.createQueue(idMinterQueue).getQueueUrl
+  def queueName: String
+
+  // Use eventually to allow some time for the local SQS to start up. Seems to be needed on CI
+  val queueUrl = eventually {
+    sqsClient.createQueue(queueName).getQueueUrl
+  }
+
+  // AWS does not delete a message automatically once it's read.
+  // It hides for the number of seconds specified in VisibilityTimeout.
+  // After the timeout has passet it will be sent again.
+  // Setting 1 second timeout for tests, to be able to test message deletion
```

Comment:
Yay! We should have a comment like this in SQSReader as well. And let’s make it clear that there’s an explicit delete step required from us. Something like:

> After a consumer reads a message from an SQS queue, AWS doesn’t delete the message immediately. It "hides" the message for a fixed period, until either
>
> 1. the consumer tells SQS to delete the message, or
> 2. the timeout expires
>
> If the timeout expires before the consumer sends a delete request, the message is unhidden and can be read by another consumer.